ray_on_golem/server/cluster/cluster.py

```diff
@@ -110,11 +109,17 @@ async def stop(self, clear: bool = True) -> None:
 
         self._state = NodeState.terminating
 
+        stop_coros = []
         for node in self._nodes.values():
-            await node.stop()
+            stop_coros.append(node.stop(call_events=False))
 
+        await asyncio.gather(*stop_coros)
```

Comment:
if we're only building the list to then call `asyncio.gather` with it, maybe it doesn't make sense to keep the intermediary list and just call the code with a list comprehension?
```suggestion
        await asyncio.gather(*[node.stop(call_events=False) for node in self._nodes.values()])
```